relate/src/main/scala/com/lucidchart/relate/SqlResult.scala

```diff
@@ -42,20 +42,16 @@ class SqlResult(val resultSet: java.sql.ResultSet) extends ResultSetWrapper with
     val mm: mutable.MultiMap[U, V] = new mutable.HashMap[U, mutable.Set[V]] with mutable.MultiMap[U, V]
     withResultSet { resultSet =>
       while (resultSet.next()) {
-        val parsed = parser(asRow)
+        val parsed = parser(SqlRow(resultSet))
```

Comment:
would it be worth replacing `asRow` with `currentRow`?

Reply:
If I did, I'd want it to be private to relate, as it doesn't really align with the rest of the public-facing interface, which all involves processing the whole `ResultSet` in one go. But really `SqlRow(resultSet)` isn't that hard to type so I think I'll just leave it